WebViewControl.Avalonia/WebView.Avalonia.cs

```diff
@@ -43,14 +43,15 @@ partial class WebView : BaseControl {
         protected override void OnGotFocus(GotFocusEventArgs e) {
             if (!e.Handled) {
                 e.Handled = true;
-                base.OnGotFocus(e);
-
-                // use async call to avoid reentrancy, otherwise the webview will fight to get the focus
-                Dispatcher.UIThread.Post(() => {
-                    if (IsFocused) {
-                        chromium.Focus();
-                    }
-                }, DispatcherPriority.Background);
+                if (!chromium.IsFocused) {
+                    base.OnGotFocus(e);
+                    // use async call to avoid reentrancy, otherwise the webview will fight to get the focus
+                    Dispatcher.UIThread.Post(() => {
```

Comment:
didnt we reach a conclusion that Dispatcher.UIThread.Post was not needed?

Reply:
It did not work when I integrated it in ServiceStudio. I had to do some small changes to get it to work

Reply:
i dont like the sound of that.... because 

if (!chromium.IsFocused) { is meant to prevent reentrancy... not the post

Reply:
Your're right. we don't need the `if (!chromium.IsFocused) ` anymore

Reply:
This does not work?
```
if (!chromium.IsFocused) {
       chromium.Focus();
 }
 `

Reply:
No, the Child window does not get focused. The main window is.